app/views/course/sections/apply.njk

```diff
@@ -4,7 +4,7 @@
     {% if course.has_vacancies %}
       {{ govukButton({
         text: "Apply for this course",
-        href: "#",
+        href: "https://www.apply-for-teacher-training.service.gov.uk/candidate/account",
```

Comment:
Since this is a prototype, I don't think you should link to the live service.

Reply:
We can link to here instead: https://apply-beta-prototype.herokuapp.com/account